### PR TITLE
Ben/docu 592

### PR DESCRIPTION
**Pull request by benradstone**
Just missing the download link update.

### CODE REVIEW

File: docs/command-reference.md [hunk 1]
```diff
@@ -738,7 +723,7 @@ OPTIONS
 #### Mandatory Parameters
 
 * **`--migration-id`** The identifier of the migration from which to remove the exclusion.
-* **`--exclusion-id`** The identifier of the exclusion to remove from the migration.
+* **`--exclusion-id`** The identifier of the exclusion to remove from the migration. This is referenced in the UI as **Name**.
```
Comment (shonaisbister-disco) [inline docs/command-reference.md:726]: Line 726: is the UI label correct? I think it's copied and pasted from above (should be ID?)

**Review by shonaisbister-disco — CHANGES_REQUESTED** [review]
Looks good, just check that one UI label in line 726.

File: docs/command-reference.md (hunk 1 above)
Comment (benradstone) [reply docs/command-reference.md:726]: exclusion_id_UI_name

It is **Name** according to the UI.

Comment (shonaisbister-disco) [reply docs/command-reference.md:726]: Perfect then! Just checking.

**Review by shonaisbister-disco — APPROVED** [review]

**Pull request merged**